spec/models/authentication_spec.rb

```diff
@@ -30,4 +30,14 @@
       expect(auth.reload.password).to eq(pwd_plain)
     end
   end
+
+  it "#retryable_status?" do
+    expect(described_class.new(:status => 'valid').retryable_status?).to       be_falsy
+    expect(described_class.new(:status => 'none').retryable_status?).to        be_falsy
+    expect(described_class.new(:status => 'incomplete').retryable_status?).to  be_falsy
+    expect(described_class.new(:status => 'error').retryable_status?).to       be_truthy
+    expect(described_class.new(:status => 'Error').retryable_status?).to       be_truthy
```

Comment:
Can you split this one out into its own paragraph?  Unlike the others testing all of the statuses, this one tests the case sensitivity.